terraform/modules/stack/services.tf

```
@@ -81,3 +82,26 @@ module "items_api" {
   deployment_service_env = var.environment_name
 }
 
+module "concepts_api" {
+  source                  = "../service"
+  service_name            = "${var.environment_name}-concepts-api"
+  deployment_service_name = "concepts-api"
+
+  container_port              = local.container_ports.concepts
+  container_image             = var.container_images.concepts
+  desired_task_count          = var.desired_task_counts.concepts
+  load_balancer_listener_port = local.concepts_lb_port
+  security_group_ids          = local.ecs_security_groups
+
+  environment = {
+    PORT = local.container_ports.concepts
+  }
+
+  secrets = {}
+
```

I assume this is true; if not, we should fix it!

```suggestion
  # Below this line is boilerplate that should be the same across
  # all Fargate services.
```